src/command/Command.php

```diff
@@ -215,19 +214,17 @@ public function setUsage(Translatable|string $usage) : void{
 	}
 
 	public static function broadcastCommandMessage(CommandSender $source, Translatable|string $message, bool $sendToSource = true) : void{
-		$users = $source->getServer()->getBroadcastChannelSubscribers(Server::BROADCAST_CHANNEL_ADMINISTRATIVE);
-		$result = KnownTranslationFactory::chat_type_admin($source->getName(), $message);
-		$colored = $result->prefix(TextFormat::GRAY . TextFormat::ITALIC);
+		$subscribers = $source->getServer()->getBroadcastChannelSubscribers(Server::BROADCAST_CHANNEL_ADMINISTRATIVE);
+		$broadcast = KnownTranslationFactory::chat_type_admin($source->getName(), $message);
 
 		if($sendToSource){
 			$source->sendMessage($message);
 		}
 
-		foreach($users as $user){
-			if($user instanceof BroadcastLoggerForwarder){
-				$user->sendMessage($result);
-			}elseif($user !== $source){
-				$user->sendMessage($colored);
+		foreach($subscribers as $user){
+			//TODO: this check is flaky, since the sender might have broadcast subscribers that we don't know about
+			if($user !== $source){
```

Comment:
This needs to be solved before merging. In the future we may not want to have players (or CommandSenders) implement this interface directly. If this happens, we'll need some other way for the sender to exclude its own broadcast receiver from receiving the message. Possibly the `CommandSender` interface could add `getChatReceiver()` or similar.